src/features/tokens/SelectOrInputTokenIds.tsx

```diff
@@ -1,65 +1,35 @@
 import { useFormikContext } from 'formik';
 
 import { TextField } from '../../components/input/TextField';
-import { AssetNamespace, getCaip19Id } from '../caip/tokens';
-import { RouteType, RoutesMap } from '../routes/types';
-import { getTokenRoute, isWarpRoute } from '../routes/utils';
 import { TransferFormValues } from '../transfer/types';
-import { useAccountAddressForChain } from '../wallet/hooks/multiProtocol';
 
 import { SelectTokenIdField } from './SelectTokenIdField';
-import { useContractSupportsTokenByOwner, useIsSenderNftOwner } from './balances';
 
-export function SelectOrInputTokenIds({
-  disabled,
-  tokenRoutes,
-}: {
-  disabled: boolean;
-  tokenRoutes: RoutesMap;
-}) {
+// import { useContractSupportsTokenByOwner, useIsSenderNftOwner } from './balances';
+
+export function SelectOrInputTokenIds({ disabled }: { disabled: boolean }) {
   const {
-    values: { originCaip2Id, tokenCaip19Id, destinationCaip2Id },
+    values: { tokenIndex },
   } = useFormikContext<TransferFormValues>();
-
-  const route = getTokenRoute(originCaip2Id, destinationCaip2Id, tokenCaip19Id, tokenRoutes);
-
-  let activeToken = '' as TokenCaip19Id;
-  if (route?.type === RouteType.CollateralToSynthetic) {
-    // If the origin is the base chain, use the collateralized token for balance checking
-    activeToken = tokenCaip19Id;
-  } else if (route && isWarpRoute(route)) {
-    // Otherwise, use the synthetic token for balance checking
-    activeToken = getCaip19Id(
-      route.originCaip2Id,
-      AssetNamespace.erc721,
-      route.originRouterAddress,
-    );
-  }
-
-  const accountAddress = useAccountAddressForChain(originCaip2Id);
-  const { isContractAllowToGetTokenIds } = useContractSupportsTokenByOwner(
-    activeToken,
-    accountAddress,
-  );
+  // const accountAddress = useAccountAddressForChain(origin);
```

Comment:
This component is used for NFT selection. We decided to leave it in place until NFT support is added to WarpCore